extensions/sql-assessment/src/assessmentResultGrid.ts

```diff
@@ -145,20 +145,31 @@ export class AssessmentResultGrid implements vscode.Disposable {
 		});
 	}
 
+	private filterOutNotSupportedKind(items: azdata.SqlAssessmentResultItem[]): azdata.SqlAssessmentResultItem[] {
+		if (this.asmtType === AssessmentType.AvailableRules) {
```

Comment:
Useful to add comments of why stuff like this is being done to give other people context of what this is for